jax/_src/random.py

```diff
@@ -1595,3 +1596,29 @@ def threefry_2x32(keypair, count):
   warnings.warn('jax.random.threefry_2x32 has moved to jax.prng.threefry_2x32 '
                 'and will be removed from `random` module.', FutureWarning)
   return prng.threefry_2x32(keypair, count)
+
+def orthogonal(
+  key: KeyArray,
+  n: int,
+  shape: Sequence[int] = (),
+  dtype: DTypeLikeFloat = dtypes.float_
+) -> jnp.ndarray:
+  """Sample uniformly from the orthogonal group O(n). If the specified
+    dtype is complex, sample uniformly from the unitary group U(n).
```

Comment:
Docstrings should start with a single summary line, and then use a new paragraph for any additional summary, aligned with the left side of the triple quote. You can see examples of this in other functions in this file, e.g.  https://github.com/google/jax/blob/0b470361dac51fb4f5ab2f720f1cf35e442db005/jax/_src/random.py#L109-L122